.github/workflows/partial-tests.yml

```diff
@@ -79,110 +78,32 @@ jobs:
 
       - name: Pytest CLI
         if: steps.changes.outputs.cli == 'true'
-        run: pipenv run pytest tests/cli
+        run: pipenv run pytest  tests/cli
 
       - name: Pytest Config
         if: steps.changes.outputs.config == 'true'
-        run: pipenv run pytest tests/config
+        run: pipenv run pytest  tests/config
 
       - name: Pytest Core
         if: steps.changes.outputs.core == 'true'
-        run: pipenv run pytest -m "not orchestrator_dispatcher and not standalone" tests/core
+        run: pipenv run pytest  tests/core
 
       - name: Pytest GUI
         if: steps.changes.outputs.gui == 'true'
-        run: pipenv run pytest tests/gui
+        run: pipenv run pytest  tests/gui
 
       - name: Pytest GUI Core
         if: steps.changes.outputs.gui-core == 'true'
-        run: pipenv run pytest tests/gui_core
+        run: pipenv run pytest  tests/gui_core
 
       - name: Pytest Logger
         if: steps.changes.outputs.logger == 'true'
-        run: pipenv run pytest tests/logger
+        run: pipenv run pytest  tests/logger
 
       - name: Pytest Rest
         if: steps.changes.outputs.rest == 'true'
-        run: pipenv run pytest tests/rest
+        run: pipenv run pytest  tests/rest
 
-      - name: Pytest Templates
+      - name: Pytest Rest
         if: steps.changes.outputs.templates == 'true'
-        run: pipenv run pytest tests/templates
-
-  submit_tests:
```

Comment:
I think we should keep this while our unit tests are flacky with the orcherstrator and standalome modes.

Reply:
Well done, I don't know how I managed to delete that code!